shotover-proxy/tests/cassandra_int_tests/functions.rs

```diff
@@ -1,19 +1,5 @@
 use crate::helpers::cassandra::{assert_query_result, run_query, CassandraConnection, ResultValue};
 
-async fn drop_function(session: &CassandraConnection) {
-    assert_query_result(
-        session,
-        "SELECT test_function_keyspace.my_function(x, y) FROM test_function_keyspace.test_function_table WHERE id=1;",
-        &[&[ResultValue::Int(4)]]
-    ).await;
-    run_query(session, "DROP FUNCTION test_function_keyspace.my_function").await;
-
-    session.execute_expect_err_contains(
```

Comment:
What if we deleted just this line, so we still get to check that "DROP FUNCTION" doesnt blow up somehow?

Reply:
Sure might as well